.github/workflows/java.yaml

```diff
@@ -0,0 +1,21 @@
+name: Java
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+  pull_request:
+    paths:
+      - "java/**"
+      - .github/workflows/java.yaml
+jobs:
+  coverage:
+    name: Test
+    runs-on: ubuntu-18.04
```

Comment:
there are three jvms on https://github.com/actions/virtual-environments/blob/ubuntu18/20210309.1/images/linux/Ubuntu1804-README.md .  It would be useful to print the JVM version that is being used for these tests.  (Under Travis, the version was explicitly being set to 1.8.0)